docs/utils.rst

```diff
@@ -67,6 +67,17 @@ Miscellaneous utility functions.
     get_data_file_path
     convert_0_1_smirnoff_to_0_2
     convert_0_2_smirnoff_to_0_3
+    attach_units
+    detach_units
+    check_units_are_compatible
+    unit_to_string
+    string_to_unit
+    convert_all_quantities_to_string
+    convert_all_strings_to_quantity
+    extract_serialized_units_from_dict
+    serialize_numpy
+    deserialize_numpy
```

Comment:
Were these just missing before, or am I forgetting a big PR in this release?

Reply:
These were missing before.